api/facadeversions.go

```diff
@@ -30,7 +30,7 @@ var facadeVersions = facades.FacadeVersions{
 	"AllWatcher":                   {3},
 	"Annotations":                  {2},
 	"Application":                  {19, 20},
-	"ApplicationOffers":            {4, 5},
```

Comment:
Just a mental note for me, we're dropping support for which version of Juju here (2.x or 3.x)?

Reply:
this is 3.x

Reply:
We need to make sure we don't leave 3.x orphaned here.